### ISSUE TITLE
Compilation toolchain for Windows

### ISSUE THREAD

**Issue by Senyoret1** [skycoin/hardware-wallet#192]
**Feature description**
Currently the emulator does not work in Windows. It would be good to make it work in that OS. However, this is not a priority, so it should not be done for V1.

**Is your feature request related to a problem? Please describe.**
At this moment I do not remember exactly the errors, since I tried it for the last time several months ago, but when trying to compile the emulator in Windows the compiler reported several errors, apparently due to elements that are not present in Windows.

**Describe the solution you'd like**
To be able to compile and run the emulator normally in Windows, as it is possible to do it in Linux.

**Describe alternatives you've considered**
Nothing for now.

**Additional context**
Nothing for now.

**Possible implementation**
Nothing for now.

**Comment by Macrosrider** [comment]
I'm almost done, but in function flash_erase_all_sectors() (in flash.c) memset() caused 'Segmantation fault' on Windows. What's also interestig, it caused only once, on first run after build, then in oledInit() (oled.c) I got error 'Failed to initialize SDL: No available video device'
